syntax = 'proto3';

package sample.v1;

service CommonService {
    rpc Ping(PingRequest) returns (PingResponse);
}

message PingRequest {
    string name = 1;
}

message PingResponse {
    string message = 1;
}